debian/control

```diff
@@ -0,0 +1,31 @@
+Source: tvb-library
+Maintainer: the virtual brain <tvb.debian@codemart.ro>
+Uploaders: umar haruna abdullahi <umarbrowser20@gmail.com>
+Section: python
+Priority: optional
+Build-Depends: debhelper (>=11),
+               dh-python,
+               python-all,
+               python-setuptools,
+               python-networkx,
+               python-numpy,
+               python-numba,
+               python-numexpr,
+               python-matplotlib,
+               python-sklearn,
+               python-scipy,
+               tvb-gdist,
+               tvb-data
+Standards-Version: 4.1.4
+Vcs-Browser: https://github.com/the-virtual-brain/tvb-library
+Vcs-Git: https://github.com/the-virtual-brain/tvb-library.git
+Homepage: http://www.thevirtualbrain.org
+
+Package: tvb-library
```

Comment:
TODO -- check how to properly name this, might want to become `python3-tvb` (`python3-tvb-library`) and the framework one could be named either just a simple `tvb` or `tvb-framework`